src/assets/stack-blitz/src/styles.scss

```diff
@@ -1,11 +1,9 @@
 body {
-  font-family: Roboto, Arial, sans-serif;
+  font-family: Roboto, "Helvetica Neue", sans-serif;
   margin: 0;
   padding: 30px;
 }
 
-.version-info {
```

Comment:
Also note that I removed this as it looks good without the extra CSS as well, and this matches more with the standard CLI project w/ `ng add @angular/material` then.